bower.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
NIT: worth syncing up some of the detail levels we've done at https://github.com/edx/ux-pattern-library/blob/master/bower.json? Namely, the license and versioning bits.